src/Semantic/Task/Files.hs

```diff
@@ -108,7 +109,7 @@ readBlobs (FilesFromHandle handle) = send (Read (FromHandle handle) pure)
 readBlobs (FilesFromPaths [path]) = do
   isDir <- isDirectory (filePath path)
   if isDir
-    then send (Read (FromDir (filePath path)) pure)
+    then send (Read (FromDir (Path.path (filePath path))) pure)
```

Comment:
We normally use `Path.parse` to have failable parsing of path types, but we know from the `isDirectory` call above that this is safe.